### DISCUSSION TITLE
can't run project with svelte on bun

### DISCUSSION THREAD

**Discussion by bobwatcherx** [oven-sh/bun#1683] [Q&A]
i can't run project svelte in my browser and in the terminal it always says 404 not found

**bun version**
root@minto-Lenovo-V145-14AST:/home/minto/belajar/sveltePro/sveltefire# bun --version
0.4.0

package.json file
```
{
  "name": "sveltefire",
  "version": "0.0.13",
  "scripts": {
    "dev": "vite dev",
    "build": "vite build",
    "preview": "vite preview",
    "check": "svelte-kit sync && svelte-check --tsconfig tsconfig.json",
    "check:watch": "svelte-kit sync && svelte-check --tsconfig tsconfig.json --watch"
  },
  "devDependencies": {
    "@sveltejs/adapter-auto": "next",
    "@sveltejs/kit": "next",
    "svelte": "^3.55.0",
    "svelte-check": "^2.10.2",
    "svelte-preprocess": "^5.0.0",
    "tslib": "^2.4.1",
    "typescript": "^4.9.4",
    "vite": "^4.0.2"
  },
  "type": "module",
  "dependencies": {
    "@bun-community/sveltekit-adapter-bun": "^0.0.6",
    "@rollup/plugin-commonjs": "^24.0.0",
    "@rollup/plugin-json": "^6.0.0",
    "@rollup/plugin-node-resolve": "^15.0.1",
    "firebase": "^9.15.0",
    "rollup": "^3.7.5"
  }
}
```

LOG MESSAGE
root@minto-Lenovo-V145-14AST:/home/minto/belajar/sveltePro/sveltefire# bun dev
[2.00ms] bun!! v0.4.0 (a94bba63)

  Link: http://localhost:3000

404 GET / as application/octet-stream

**Comment by hasanyasin** [comment]
It should be `bun run dev`, not `bun dev` since `bun dev` is already a bun command and here what you want is to call the "dev" script defined in package.json.

**Reply by deniskipeles** [reply]
### Hello?
And thank you for your suggestion! In most cases, running "bun run dev" might be the correct approach. However, I'd like to clarify that I am using SvelteKit, and it seems that the "bun run dev" command is not working as expected in my specific context.

In the case of SvelteKit, the recommended development command is typically different. To start the development server in SvelteKit, I usually use:

```bash
npm run dev

**Reply by hasanyasin** [reply]
Bun currently supports Vite and Svelte-Kit. I don't have `npm` on my dev machines anymore.
